ivy/functional/backends/numpy/creation.py

```diff
@@ -137,7 +137,11 @@ def full_like(
     device: str,
     out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
+ numpy_sort
+    # _assert_fill_value_and_dtype_are_compatible(dtype, fill_value)
+
     ivy.assertions.check_fill_value_and_dtype_are_compatible(fill_value, dtype)
+ master
```

Comment:
Hi @Hellenn07, can you please take a look at these lines, it appears the merge from master left some text that shouldn't be in the code. Kindly fix these and request for a review once you are done thanks! 🙂